src/components/SentimentAnalysisGraph.js

```diff
@@ -0,0 +1,50 @@
+import React, { Component } from 'react';
+import { VictoryAxis } from 'victory';
+import { VictoryBar } from 'victory';
+import { VictoryChart } from 'victory';
+
+class SentimentAnalysisGraph extends Component {
+	static displayName = 'SentimentAnalysisGraph';
+
+	state = {
+		callFunc: this.getData(),
+		data: {pos_value: 90, neg_value: 10}
+	};
+
+	getData(){
+		setInterval(() => {
+			const rand = Math.floor(Math.random() * 100);
+			this.setState({
+				data:{pos_value: rand,
+					neg_value: 100 - rand}
+			});
+		}, 2500);
+	}
+
+	render() {
+		return (
+			<VictoryChart domain={{y: [0, 100]}} height={220}>
+				<VictoryAxis
+					style={{
+						axisLabel: {fontSize: 25}}}
+					tickValues={[' ', '', 'Positive', 'Negative', '', '']}
+					label = "Sentiment Anaylysis Report"/>
+
+				<VictoryBar
+					style={{
+						data: {width: 30},
+						labels: {fontSize: 7}
```

Comment:
`width` and `fontSize` should also be either props or constants declared above the class declaration